### PR TITLE
[REF] enforce hard wrapping and remove abbreviation

### PR DESCRIPTION
**Pull request by Remi-Gau**
- [x] replaces occurrences of BSM 
- [x] remove latin abbreviations
- [x] apply hard word wrapping to all markdown

### CODE REVIEW

File: specification/index.md [hunk 1]
```diff
@@ -1,32 +1,39 @@
...
+:::{panels} :column: col-lg-6 px-2 py-2
 
-:::{panels}
-:column: col-lg-6 px-2 py-2
 ---
-**📖 Learn more**
-^^^
-**[](motivation.md)**: Why do we need *BSM*?
+
+**📖 Learn more** ^^^ **[](motivation.md)**: Why do we need _BIDS Stats Models_?
```
Comment (Remi-Gau) [inline specification/index.md:22]: seems the auto lint may have messed up a few things

let me cross check and fix

File: specification/legacy.md [hunk 2]
```diff
@@ -85,52 +81,54 @@ in a group column of a participants.tsv file:
...
 
 ```yaml
 {
   "Name": "my_analysis",
   "BIDSModelVersion": "1.0.0",
-  "Input": {
-    "task": "motor",
-    "run": 1
-  },
+  "Input": { "task": "motor", "run": 1 },
```
Comment (Remi-Gau) [inline specification/legacy.md:106]: prettier seems to have reformatted some of those too on top of the word wrapping

File: specification/index.md (hunk 1 above)
Comment (Remi-Gau) [reply specification/index.md]: fixed

File: specification/default_model.md [hunk 3]
```diff
@@ -7,10 +7,10 @@ See below for code snippet showing how to do it with each package.
 
 ## PyBIDS - `auto_model`
 
-[PyBIDS](https://github.com/bids-standard/pybids) currently represents
-models as dictionaries.
-To generate a default model dictionary, use {py:func}`bids.modeling.auto_model`,
-and then use {py:func}`json.dump` to save it to a file.
+[PyBIDS](https://github.com/bids-standard/pybids) currently represents models as
+dictionaries. To generate a default model dictionary, use
```
[the remark this answers is not in the document]
Comment (Remi-Gau) [reply specification/default_model.md:11]: I can revert the word wrapping if you want

**Comment by adelavega** [comment]
@Remi-Gau what's the status of this? merge conflict?

**Comment by Remi-Gau** [comment]
expect a force push on this one:
- will reset
- will redo the remove BSM abbreviation
- implement semantic line break

File: specification/default_model.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Remi-Gau) [reply specification/default_model.md]: reverted

**Comment by Remi-Gau** [comment]
> @Remi-Gau what's the status of this? merge conflict?

should be fixed now

**Pull request merged**